Sprint10/Tema3.js

```diff
@@ -0,0 +1,17 @@
+// Scrieți o funcție pentru a găsi numărul de apariții al unei litere într-un șir
+
+const sir = "Vreau sa lucrez in IT";
+litera = "o"
```

Comment:
```suggestion
const litera = "o"
```

Desi functioneaza si fara, e bine sa ne obisnuim sa declaram orice variabila cu "let" sau "const"